health.go

```diff
@@ -487,6 +489,33 @@ func readIntFromFile(filePath string) (num int, err error) {
 	return strconv.Atoi(strings.TrimSpace(string(data)))
 }
 
+func getTHPConfigs() map[string]interface{} {
```

Comment:
Any reason for not `map[string]string`?... At least then it can't be "anything".

Reply:
One of the configs being captured (`max_ptes_none`) is an integer, so thought of keeping it flexible. We can of course store the integer also in its string form and use `Atoi` when reading it in `health-analyzer` if required. Would that be a more preferred approach?

Reply:
Now we can return `map[string]string` from function.

Reply:
ok, will change accordingly.